packages/apps/kadena-docs/src/components/Layout/components/SideMenu/MenuCard.tsx

```diff
@@ -8,12 +8,14 @@ export interface IMenuCardProps {
   idx: number;
   ref?: React.ForwardedRef<HTMLDivElement>;
   onClick?: MouseEventHandler<HTMLUListElement>;
+  cy?: string;
 }
 
 export const MenuCard: FC<IMenuCardProps> = React.forwardRef(
-  ({ children, onClick, active, idx = 0 }, ref) => {
+  ({ children, onClick, active, idx = 0, cy }, ref) => {
```

Comment:
Maybe make this more explicit, like dataCy? For a second I thought we were passing some kind of cypress instance here